examples/decoding/plot_receptive_field.py

```diff
@@ -96,16 +104,16 @@
 
 # Prepare model data (make time the first dimension)
 speech = speech.T
-Y, _ = raw[:]  # Outputs for the model
-Y = Y.T
+EEG, _ = raw[:]  # Outputs for the model
+EEG = EEG.T
```

Comment:
please don't rename variables as PR can become controversial for little gain.

we usually use data (not EEG) and here it is written Y as it is the standard name for predicted target variable in ML

Reply:
OK. The EEG is not the predicted variable in the backward case so I thought it'd make more sense.